ups/python.cfg

```diff
@@ -11,19 +12,30 @@ class Configuration(lsst.sconsUtils.Configuration):
     def __init__(self):
         self.name, self.root = self.parseFilename(__file__)
 
+    @staticmethod
+    def _get_config_var(name):
+        """The relevant Python is not guaranteed to be the Python
+        that we are using to run SCons so we must shell out to the
+        PATH python."""
+        pycmd = 'import distutils.sysconfig as s; print(s.get_config_var("{}"))'.format(name)
+        result = subprocess.check_output(["python", "-c", pycmd]).decode().strip()
```

Comment:
You mean an error from decoding the bytes? SCons will be the thing the dies. Even if we caught it and returned None the compilation would break shortly afterwards.